CHANGES.md

```diff
@@ -1,6 +1,11 @@
 # Unreleased
 
-- [NEW] Add `CloudantClient.metaInformation()`.
+- [NEW] Add `CloudantClient.metaInformation()` to get meta information from the welcome page.
+- [NEW] Add methods for interacting with the replicator's `_scheduler` endpoint:
+  - `CloudantClient.metaInformation()`,
```

Comment:
Don't think `metaInformation` is needed here as it is called out on the line above (and unrelated to the `_scheduler` endpoint).

Reply:
Good point, must have been a sloppy merge. Fixed in 5303fc7